packages/components/Button/src/FAB/FAB.test.tsx

```diff
@@ -11,32 +18,48 @@ beforeAll(() => {
 });
 
 it('Default FAB (iOS)', () => {
-  const tree = renderer.create(<FAB>Default FAB (iOS)</FAB>).toJSON();
+  const tree = renderer.create(<FAB icon={{ svgSource: svgProps }}>Default FAB (iOS)</FAB>).toJSON();
```

Comment:
>icon={{ svgSource: svgProps }} [](http://example.com/codeflow?start=36&length=30)

Why are we adding icons to all the test cases? Is it needed for something?

Reply:
FAB cannot exist without an icon, have made the prop a required one